test/suites/proxy.sh

```diff
@@ -47,3 +54,166 @@ test_proxy_device() {
   lxc delete -f proxyTester
 }
 
+test_proxy_device_unix() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${LXD_ADDR}"
+
+  MESSAGE="Proxy device test string"
+  OUTFILE="$(pwd)/proxyTest.out"
+  HOST_SOCK="$(pwd)/host.sock"
```

Comment:
Uhm, if you run that in parallel on the same builder then you'd have a conflict, won't you?